crates/sui-protocol-config/src/lib.rs

```diff
@@ -1469,6 +1479,12 @@ impl ProtocolConfig {
                     cfg.max_jwk_votes_per_validator_per_epoch = Some(240);
                     cfg.max_age_of_jwk_in_epochs = Some(1);
                 }
+                26 => {
+                    // zklogin::check_zklogin_id
+                    cfg.check_zklogin_id_cost_base = Some(200);
+                    // zklogin::check_zklogin_iss
+                    cfg.check_zklogin_iss_cost_base = Some(200);
```

Comment:
cc @dariorussi for gas costs

Reply:
don't know it seems fine to me to go in and we revisit as we go.
Adding @oxade as well given he may have more of an intuition for those numbers in native functions